_overviews/contributors/index.md

```diff
@@ -0,0 +1,809 @@
+and shows how a typical development environment looks like.
+
+An example of library that follows the recommendations given here is available at
+[https://github.com/scalacenter/library-example](https://github.com/scalacenter/library-example). For the sake
+of conciseness, this example uses mainstream technologies like GitHub, Travis CI, and sbt, but alternative
+technologies will be mentioned and adapting the contents of this document for them should be straightforward.
+
+## Choose an Open Source License
+
+The first step consists in choosing an open source license specifying under which conditions the library
+can be reused by other people. You can browse the already existing open source licenses on the
+[opensource.org](https://opensource.org/licenses) website. If you don’t know which one to pick, we suggest
+using the [Apache License 2.0](https://opensource.org/licenses/Apache-2.0), which allows users to use (including
+commercial use), share, modify and redistribute (including under different terms) your work under the condition
+that the license and copyright notices are preserved.
+
+Once you have chosen a license, *apply* it to your project by creating a `LICENSE` file in the root directory
+of your project with the license contents or a link to it. This file usually indicates who owns the copyright.
+We suggest putting the project “contributors” as copyright owners. You can see our
+[LICENSE file](https://github.com/scalacenter/library-example/blob/master/LICENSE) as an example.
+
+## Host the Source Code
+
+We recommend to share the source code of your library by hosting it on a public [Git](https://git-scm.com/)
+repository such as [GitHub](https://github.com), [Bitbucket](https://bitbucket.org) or [GitLab](https://gitlab.com).
+In our example, we use GitHub.
+
+Your project should include a [README](https://github.com/scalacenter/library-example/blob/master/README.md) file
+including a description of what the library does and some documentation (or links to the documentation).
+
+You should take care of putting only source files under version control. For instance, artifacts generated by the
+build system should *not* be versioned. You can instruct Git to ignore such files by adding them to a
+[.gitignore](https://github.com/scalacenter/library-example/blob/master/.gitignore) file.
+
+In case you are using sbt, make sure your repository has a
+[project/build.properties](https://github.com/scalacenter/library-example/blob/master/project/build.properties)
+file indicating the sbt version to use, so that people (or tools) working on your repository will automatically
+use the correct sbt version.
+
+## Setup Continuous Integration
+
+The first reason for setting up a continuous integration (CI) server is to systematically run tests on pull requests.
+Examples of CI servers are [Travis CI](https://travis-ci.org), [Drone](https://drone.io) or
```

Comment:
maybe insert "that are free for open source projects" — if that's actually true of Drone, I don't know if it is?